src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -80,16 +80,16 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.debug')->addTag('twig.extension');
         }
 
-        $composerRootDir = $this->getComposerRootDir($container->getParameter('kernel.root_dir'));
+        $projectDir = $container->getParameter('kernel.project_dir');
         $twigLoader = $container->getDefinition('twig.loader.native_filesystem');
         if ($container->has('templating')) {
             $loader = $container->getDefinition('twig.loader.filesystem');
             $loader->setMethodCalls(array_merge($twigLoader->getMethodCalls(), $loader->getMethodCalls()));
-            $loader->replaceArgument(2, $composerRootDir);
+            $loader->replaceArgument(2, $projectDir);
```

Comment:
That's what I thought in the very first place, but that's conditional and depending on in the if or the else branch, the argument number is not the same, that's why I didn't do this without being aware of the full impact this would have.